app/assets/stylesheets/publisher.scss

```diff
@@ -243,14 +237,6 @@
     .hide-location { display: inline-block; }
     .locator { display: none; }
   }
-
-  .twitter-typeahead {
-    left: -1px;
-    // Override inline rule of Typeahead
-    // scss-lint:disable ImportantRule
-    position: absolute !important;
```

Comment:
If I keep this line, width for the comment publisher gets improper:

<img width="960" alt="2017-03-23 19-12-33" src="https://cloud.githubusercontent.com/assets/10187586/24260595/1cdaf126-0ffd-11e7-919c-3fa5dd4ed982.png">

However I don't see any issues without this line in the main publisher (exacly like at your screenshot):

<img width="960" alt="2017-03-23 19-18-30" src="https://cloud.githubusercontent.com/assets/10187586/24260798/ac9fb896-0ffd-11e7-9db9-66621487c414.png">

So what's wrong with the CSS I proposed there?